source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewController.mm

```diff
@@ -225,20 +226,25 @@ - (void) addTasksToConcurrentQueue:(std::vector<std::shared_ptr<BaseCardElement>
                 [self tagBaseCardElement:elem];
                 /// dispatch to concurrent queue
                 std::shared_ptr<TextBlock> txtElem = std::dynamic_pointer_cast<TextBlock>(elem);
-                ACOHostConfig *acoConfig = [[ACOHostConfig alloc] initWithConfig:_hostConfig];
                 dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0),
                     ^{
                         std::string dateParsedString = [ACOHostConfig getLocalizedDate:txtElem];
                         // MarkDownParser transforms text with MarkDown to a html string
                         std::shared_ptr<MarkDownParser> markDownParser = std::make_shared<MarkDownParser>(dateParsedString.c_str());
                         NSString *parsedString = [NSString stringWithCString:markDownParser->TransformToHtml().c_str() encoding:NSUTF8StringEncoding];
-
+                        // if correctly initialized, fonFamilyNames array is bigger than zero
+                        NSMutableString *fontFamilyName = [[NSMutableString alloc] initWithString:@"'"];
+                        for(NSUInteger index = 0; index < [_hostConfig.fontFamilyNames count] - 1; ++index){
+                            [fontFamilyName appendString:@"', '"];
+                        }
+                        [fontFamilyName appendString:_hostConfig.fontFamilyNames[[_hostConfig.fontFamilyNames count] - 1]];
```

Comment:
>[fontFamilyName appendString:_hostConfig.fontFamilyNames[[_hostConfig.fontFamilyNames count] - 1]]; [](start = 24, length = 99)

You have to append the font family names to the string inside the loop.

Reply:
thanks

---
In reply to: [172351182](https://github.com/Microsoft/AdaptiveCards/pull/1207#discussion_r172351182) [](ancestors = 172351182)